init-mqtt.h

```diff
@@ -129,15 +129,13 @@ void initMqtt() {
 }
 
 void publishInt(const char* topic, int d) {
-  char v[5];
-  sprintf(v, "%d", d);
-
+  
   DynamicJsonBuffer jsonBuffer;
   JsonObject& json = jsonBuffer.createObject();
 
   json[PAYLOAD_DEVICE] = qrConfig.DEVICE_ID;
   json[PAYLOAD_TOPIC] = topic;
-  json[PAYLOAD_VALUE] = v;
+  json[PAYLOAD_VALUE] = d;
   char payload[100];
   json.printTo(payload, sizeof(payload));
```

Comment:
as payload is char[100] - sizeof will always return 100.
Maybe it is better idea to use strlen() function for strings.

Reply:
Agree
Will look there